build_runner/test/server/serve_handler_test.dart

```diff
@@ -223,8 +225,8 @@ void main() {
     });
 
     group('WebSocket handler', () {
-      BuildUpdatesWebSocketHandler buildUpdatesWebSocketHandler;
-      Function createMockConection;
+      BuildUpdatesWebSocketHandler handler;
+      Future<Null> Function(WebSocketChannel, String) createMockConnection;
```

Comment:
[nit] Prefer `Future<void>` over `Future<Null>`. The reason you see `Future<Null>` in so much existing code is `void` used to not work there, but now that it does it's our preference for new code